openvpn/files/client.jinja

```diff
@@ -2,8 +2,6 @@
 # Managed by Salt
 # Template: {{ source }}
 
-client
```

Comment:
You're making a good point in removing `client`. But on the other hand I fear existing setups will break if they haven't set `pull` in their Pillar.
So I suggest adding a big warning to `README.rst`. What do you think?

Reply:
Yea, I agree. Breaking it wouldn't be good.. the big warning sounds good.

Reply:
On second thought.. the "big warning" is worth nothing if you force the `client` option on the user anyway. In the current state, there's no way to run an OpenVPN client without the `client` option.

So what would the warning read? "please don't use this formula to run a vpn client if you don't trust the server to have control over the client's routing table."

That's a bit weird.

Seeing that it's best practice and **strongly recommend** to fork formulas anyway (https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html) and this is a security issue, I would rather break it for users who pull from github directly (which is stupid anyways) than not allowing security conscious users to use this formula without altering it.

Reply:
@marconett I fully agree. And I wasn't being clear. Sry. I am _for_ removing `client`. Security trumps convenience.

The warning would read something like: _"We removed `client` from the client config. If you want your clients to accept `push` from the server, set `pull: True` in your Pillar."_ It is intended to warn users who are doing an update of the formula.

What do you think?

Reply:
Sounds good.